app/models/user.rb

```diff
@@ -52,7 +52,11 @@ def self.from_social_provider(provider, user_params)
   def full_name
     return username if first_name.blank?
 
-    "#{first_name} #{last_name}"
+    if Flipper.enabled?(:test_feature)
+      "#{last_name} #{first_name}"
+    else
+      "#{first_name} #{last_name}"
+    end
   end
```

Comment:
I'll remove this change before merging the PR, this is just a small POC within the PR